import_export_google_civic/controllers.py

```diff
@@ -659,6 +661,7 @@ def store_one_ballot_from_google_civic_api(one_ballot_json, voter_id=0, polling_
                     voter_address_dict,
                     election_date_text, election_description_text,
                     google_civic_election_id, state_code, voter_id, '')
+                # TODO: 6/25/17, If we store a ballot without latidude and longitude, it will not be found by a geo search.  Is this intentional?  Does the data have value?
```

Comment:
We need to save ballot_returned entries even if we don' t have lat/long because we can exceed our Google API limits. We may also not want to "burn through" our available geocode API calls. That is why we don't always try to geocode when we don't have a latitude/longitude.